src/app/model/schema.ts

```diff
@@ -215,7 +216,7 @@ export interface MissionInterface {
   deliveryLocation: Location; // default to recipient location
   deliveryConfirmationImage: string;
   deliveryNotes: string;
-  deliveryType: string;
+  deliveryType: "delivery" | "curbside";
```

Comment:
:eyes:

Reply:
its like a poor mans enum. Really nice cause it auto fills these options when either filling the prop or using it to compare

Reply:
Might as well actually define String Enum lol.... Cuz then we can share and reuse elsewhere when needed. Don't want to give people reason to cooy-pasta.

Reply:
Haha okay. Fair enough

Reply:
I threw the eyes in because this was changed from `"delivery" | "curbside"` to `string` a few days ago and now back to this lol

Reply:
Oh lol...
It should definitely be a limited set of values as we're using it to determine UI state.